juriscraper/opinions/united_states/state/ill.py

```diff
@@ -28,7 +34,7 @@ def __init__(self, *args, **kwargs):
             r"(?P<year>\d{4})? ?"
             r"(?P<court>(IL App)|(IL)) "
             r"(\(?(?P<district>[1-5])\w{1,2}\))? ?"
-            r"(?P<docket>\d{5,10})-?U?[BCD]?"
+            r"(?P<docket>\d{5,10}\w{1,2})-?U?[BCD]?"
         )
```

Comment:
can you point me to some Supreme Court opinions that have need the change 

and can you point me to the opinions with U B C and D.  I went to the website but I can't find any.  Im sure they exist, but I'm not seeing any.

Reply:
**2020 IL App (2d) 190759WC-U**
![Screenshot from 2022-01-04 11-01-17](https://user-images.githubusercontent.com/8507838/148096385-ee3dea28-0c30-499a-8304-ca472c2b2ced.png)

Reply:
**2022 IL App (2d) 120444-B**
![Screenshot from 2022-01-21 12-01-35](https://user-images.githubusercontent.com/8507838/150577337-e86eb6df-2949-4c74-99d5-98edd47b9273.png)

Reply:
**2021 IL App (4th) 180498-UB**
![Screenshot from 2022-01-21 12-03-46](https://user-images.githubusercontent.com/8507838/150577584-80d238e0-0451-433e-9a3d-17a9d5612a4d.png)

Reply:
**2021 IL App (3d) 130190-C**
![Screenshot from 2022-01-21 12-05-01](https://user-images.githubusercontent.com/8507838/150577773-8bb0bf5c-6f53-4588-bb95-34055432b104.png)